nbs/models.tft.ipynb

```diff
@@ -15,20 +15,14 @@
    "source": [
```

Comment:
<p>The equation is not compiling here. It looks like this: </p><p> </p><p>\begin{align} \eta_{1} &amp;= \mathrm{ELU}(\mathbf{W}<em>{1}\mathbf{a}+\mathbf{W}</em>{2}\mathbf{c}+\mathbf{b}<em>{1}) \ \eta</em>{2} &amp;= \mathbf{W}<em>{2}\eta</em>{1}+b_{2} \ \mathrm{GRN}(\mathbf{a}, \mathbf{c}) &amp;= \mathrm{LayerNorm}(a + \textrm{GLU}(\eta_{2})) \end{align}</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/293/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>